src/styles/sizing.css

```diff
@@ -1,7 +1,9 @@
 :root {
   --input-height: 38px;
   --input-icon-size: 16px;
+  /* TODO: remove the need for --navigation-header-height
```

Comment:
As this comment mentions, the one thing I think is objectively worse about the `body`-as-scrolling layout approach is that it requires we know the height of the sticky nav in order to implement the sidebar behaviour design has given us`*`. The simple-but-not-robust solution to that problem is to have hard-coded CSS vars as we have here.

This hard-coded approach kind of forces us to make our layouts brittle in ways that will affect users who change their default font-size. I think a more robust solution here would be to observe the sticky bar height and dynamically update the calculated height, to avoid that brittleness (as mentioned in the `sidebar-sidecar-layout/style.module.css`).

But, holding off on the not-hard-coded enhancement for now - want to get a higher-level go-ahead that this layout approach makes sense first! 🙇 

> `*` CAVEAT - seems required as far as I can tell - we had to run this kind of layout in Learn and ran into a very similar conundrum... in that case we've just stuck with magic static numbers for the nav bar heights, I think, but it's always kinda worried me.